exporter/clickhouseexporter/exporter_traces_test.go

```diff
@@ -77,9 +59,15 @@ func newTestTracesExporter(t *testing.T, dsn string, fns ...func(*Config)) *trac
 func simpleTraces(count int) ptrace.Traces {
 	traces := ptrace.NewTraces()
 	rs := traces.ResourceSpans().AppendEmpty()
+	rs.SetSchemaUrl("https://opentelemetry.io/schemas/1.4.0")
+	rs.Resource().SetDroppedAttributesCount(10)
+	rs.Resource().Attributes().PutStr("service.name", "test-service")
 	ss := rs.ScopeSpans().AppendEmpty()
 	ss.Scope().SetName("io.opentelemetry.contrib.clickhouse")
 	ss.Scope().SetVersion("1.0.0")
+	ss.SetSchemaUrl("https://opentelemetry.io/schemas/1.7.0")
+	ss.Scope().SetDroppedAttributesCount(20)
+	ss.Scope().Attributes().PutStr("lib", "clickhouse")
```

Comment:
Can we see assertions showing this data is indeed being ingested?

Reply:
done

Reply:
BTW, are we need to add new columns for other fields, like `InstrumentationScope.Attributes`, `InstrumentationScope.DroppedAttributesCount`, etc?

Reply:
> BTW, are we need to add new columns for other fields, like InstrumentationScope.Attributes, InstrumentationScope.DroppedAttributesCount, etc? 

The scope attributes are not widely adopted in the OTel project, so they will likely be empty most of the time, but it may be reasonable to add them to avoid future breaking changes. Feel free to submit another PR